fuse_msgs/CMakeLists.txt

```diff
@@ -1,26 +1,33 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.5)
 project(fuse_msgs)
 
-find_package(catkin REQUIRED COMPONENTS
-  message_generation
-  std_msgs
-)
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
```

Comment:
[C++17 in ROS Rolling](https://www.ros.org/reps/rep-2000.html#rolling-ridley-june-2020-ongoing)

Reply:
Ahh, let me change it

Reply:
https://github.com/locusrobotics/fuse/compare/5c22c91210203a8b48afe2aaf01409c1c8aa8626..5c8940af69c840a6f7fe4a16af9a7420be658ccd